sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/AmqpCoordinatorLink.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.implementation;
+
+import com.azure.core.amqp.AmqpLink;
+import org.apache.qpid.proton.amqp.transport.DeliveryState;
+import reactor.core.publisher.Mono;
+
+import java.nio.ByteBuffer;
+
+/**
+ * This is AMQP coordinator link which allow operations related to transactions.
+ */
+public interface AmqpCoordinatorLink extends AmqpLink {
```

Comment:
Do we need this to live in implementation. Is there a way we can create "DeliveryStates" that aren't used by proton-j and map them internally? I'm trying to decrease the number of classes that live in implementation that we'll have to pull out later.